schedule/yast/select_modules_and_patterns+registration/select_modules_and_patterns+registration.yaml

```diff
@@ -10,44 +10,31 @@ description: >
         report any issues;
```

Comment:
Looks like a modification of select_modules_and_patterns+registration.yaml, but it's a merge of select_modules_and_patterns+registration_x86_64.yaml and select_modules_and_patterns+registration_aarch64.yaml